app/components/enrolment_confirmation_component/enrolment_confirmation_component.html.erb

```diff
@@ -0,0 +1,28 @@
+<% if @current_user %>
+  <% if @programme.enrolment_confirmation_required? %>
+    <div class="enrolment-confirmation-component">
+      <%= render ModalComponent.new(
+        title: "Enrol on the #{@programme.certificate_name}",
+        reopen_button_text: (@button_text || "Enrol"),
+        show_corner_decoration: false,
+        reopen_button_class: ("enrolment-confirmation-component__button--full-width" if @full_width)
+        ) do |component| %>
+        <% component.with_body do %>
+          <div class="enrolment-confirmation-component__wrapper">
+            <div>
+              <h2 class="govuk-heading-s">Ready to enrol?</h2>
+              <p class="govuk-body">Confirm your enrolment on the certificate, and then let"s find you the perfect training course to get started with.</p>
+            </div>
+            <div>
+              <%= link_to "Comfirm my enrolment", @programme.enrol_path(user_programme_enrolment: { user_id: @current_user.id, programme_id: @programme.id }), method: :post, class: "govuk-button button", role: "button", draggable: "false" %>
+            </div>
+          </div>
+        <% end %>
+      <% end %>
+    </div>
+  <% else %>
```

Comment:
We have nothing in here to deal with a logged in user who is already enrolled?

I think current versions swap to a "view my dashboard" button when they are already enrolled?

Reply:
Good spot, I didn't consider that scenario. I'll double check what we currently do and add in some logic to handle that.